integration_tests/custom_reporters/__tests__/add-fail-test.js

```diff
@@ -0,0 +1,9 @@
+const add = require('../add');
+
+describe('CustomReporters', () => {
+    test('adds fail', () => {
```

Comment:
we use 2 spaces for indentation. Can you run `yarn prettier` to format the code?